containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/mysql-client-install.sh

```diff
@@ -23,9 +23,9 @@ TARBALL_URL=https://github.com/ddev/mysql-client-build/releases/download/${TARBA
 
 # Install the related mysql client if available
 set -x
-cd /tmp && curl -L -o /tmp/mysql.tgz --fail -s ${TARBALL_URL}
+cd /tmp && timeout 30 curl -L -o /tmp/mysql.tgz --fail -s ${TARBALL_URL}
```

Comment:
We may want to consider using curl's built-in timeout options for these cases in the future, https://unix.stackexchange.com/questions/94604/does-curl-have-a-timeout

Reply:
Yes, we can use the curl timeout for cases like this, but I'm very confused by having multiple different timeouts in curl. So I'll leave it as it is.